tox.ini

```diff
@@ -24,11 +24,26 @@ deps =
     andi==0.4.1
     attrs==21.3.0
     parsel==1.5.0
-    scrapy==2.6.0
```

Comment:
Why is min scrapy version removed here? I think we should be running at least one test with the minimum Scrapy version which we support.

Reply:
addressed in d17adcb